common/worldFactory.h

```diff
@@ -1,18 +0,0 @@
-#ifndef WorldFactory_h
```

Comment:
Define important? If you want it back, make a pull request where it is integrated with the project.